### ISSUE TITLE
jsx-no-undef doesn't catch lowercase object access

### ISSUE THREAD

**Issue by SystemParadox** [yannickcr/eslint-plugin-react#435]
Further to #172, jsx-no-undef does not catch lowercase object access.

This warns as expected:

```
<Foo.Bar />
```

This does not:

```
<foo.Bar />
```

Presumably it's being excluded by the code which ignores builtin tags such as `<div>`, etc. Object access should always be treated as a variable reference, regardless of case.

Thanks very much.

**Comment by SystemParadox** [comment]
Excellent. Thanks very much.
